Features/src/features/spatial/SpatialFeature.java

```diff
@@ -51,6 +51,7 @@
 	}
 
 	//-------------------------------------------------------------------------
+    private static final int[] NULL_POSITION = new int[]{-1};
```

Comment:
I couldn't find which version this was using, and since the change was trivial decided to use this instead